packages/webdriver/src/request.ts

```diff
@@ -98,36 +119,43 @@ export default class WebDriverRequest extends EventEmitter {
         /**
          * if the environment variable "STRICT_SSL" is defined as "false", it doesn't require SSL certificates to be valid.
          */
-        requestOptions.https.rejectUnauthorized = !(
+        requestOptions.https!.rejectUnauthorized = !(
             process.env.STRICT_SSL === 'false' ||
             process.env.strict_ssl === 'false'
         )
 
         return requestOptions
     }
 
-    async _request (fullRequestOptions, transformResponse, totalRetryCount = 0, retryCount = 0) {
-        log.info(`[${fullRequestOptions.method}] ${fullRequestOptions.uri.href}`)
+    private async _request (
+        fullRequestOptions: got.Options,
+        transformResponse?: (response: got.Response, requestOptions: got.HTTPSOptions) => got.Response,
+        totalRetryCount = 0,
+        retryCount = 0
+    ): Promise<WebDriverResponse> {
+        log.info(`[${fullRequestOptions.method}] ${(fullRequestOptions.url as URL).href}`)
 
         if (fullRequestOptions.json && Object.keys(fullRequestOptions.json).length) {
             log.info('DATA', transformCommandLogResult(fullRequestOptions.json))
         }
 
+        let response = await got.default(fullRequestOptions.url!, { ...fullRequestOptions })
+            // @ts-ignore
```

Comment:
let's avoid using `@ts-ignore` where possible

Reply:
I would like to create an issue for this because I am not sure if this `catch` method is even useful here. Don't want to modify the current implementation.

Reply:
what error are you getting here? There should be no errors and no need to use `ts-ignore` I guess, similar to https://github.com/webdriverio/webdriverio/blob/master/packages/wdio-shared-store-service/src/client.ts#L15

Reply:
I am getting:

```js
Property 'catch' does not exist on type 'CancelableRequest<unknown> | Request'.
  Property 'catch' does not exist on type 'Request'.ts(2339)
```

Reply:
It's happening because of the way we pass options, it's safe to ignore